test-lib/Foo.hs

```diff
@@ -0,0 +1,13 @@
+module Foo
```

Comment:
Just some dummy modules for now. We could expand it to something more complex to test various edge case.